site/content/docs/4.3/components/carousel.md

```diff
@@ -208,6 +208,54 @@ Add `data-interval=""` to a `.carousel-item` to change the amount of time to del
 </div>
 {{< /example >}}
 
+### Changing colors
+
+Change the color of a carousel's next and previous controls, indicators, and captions with a single `.text-*` utility on the `.carousel`. Need more control? Apply text utilities and custom styles more selectively.
+
+{{< example >}}
+<div id="carouselExampleColors" class="carousel slide text-white" data-ride="carousel">
+  <ol class="carousel-indicators">
+    <li data-target="#carouselExampleColors" data-slide-to="0" class="active"></li>
+    <li data-target="#carouselExampleColors" data-slide-to="1"></li>
+    <li data-target="#carouselExampleColors" data-slide-to="2"></li>
+  </ol>
+  <div class="carousel-inner">
+    <div class="carousel-item active">
+      {{< placeholder width="800" height="400" class="bd-placeholder-img-lg d-block w-100" color="#555" background="#777" text="First slide" >}}
+      <div class="carousel-caption d-none d-md-block">
+        <h5>First slide label</h5>
+        <p>Nulla vitae elit libero, a pharetra augue mollis interdum.</p>
+      </div>
+    </div>
+    <div class="carousel-item">
+      {{< placeholder width="800" height="400" class="bd-placeholder-img-lg d-block w-100" color="#444" background="#666" text="Second slide" >}}
+      <div class="carousel-caption d-none d-md-block">
+        <h5>Second slide label</h5>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+      </div>
+    </div>
+    <div class="carousel-item">
+      {{< placeholder width="800" height="400" class="bd-placeholder-img-lg d-block w-100" color="#333" background="#555" text="Third slide" >}}
+      <div class="carousel-caption d-none d-md-block">
+        <h5>Third slide label</h5>
+        <p>Praesent commodo cursus magna, vel scelerisque nisl consectetur.</p>
+      </div>
+    </div>
+  </div>
+  <a class="carousel-control-prev" href="#carouselExampleColors" role="button" data-slide="prev">
```

Comment:
`<a role="button">`, looks nasty. Can we just use `<button>`s here?

Reply:
This is how the carousel works today, so this isn't any kind of regression. I'm down to use buttons here, but I'd need help to rewrite the JS that attaches these anchors to the appropriate carousel.